avocado/utils/kernel.py

```diff
@@ -85,7 +89,7 @@ def configure(self):
         Configure/prepare kernel source to build.
         """
         self.linux_dir = os.path.join(self.work_dir, 'linux-%s' % self.version)
-        build.make(self.linux_dir, extra_args='O=%s mrproper' % self.build_dir)
+        build.make(self.linux_dir, extra_args='-C %s mrproper' % self.linux_dir)
```

Comment:
It looks like this is actually fixing a bug.  Did you have success building a kernel with the current version of this library?

Reply:
Yes, I tested that I could successfully build the kernel with the current version of this library.

Reply:
OK... please forgive me, but would you please elaborate on the reason for this change? I'm confused because `build.make()` will `chdir()` into `self.linux_dir` anyway, so `-C` + `self.linux_dir` as an argument to `make` seems to unnecessary.  The only change I can see here is that before this change, the build output would go to `build_dir` and now it seems that it'll go to `linux_dir`, but I don't get why this is necessary or a positive change.